scripts/overlay-generator

```diff
@@ -224,24 +231,17 @@ src_compile() {
 }
 
 src_install() {
+    if test "\${VCS_HOME}" != ""
+    then
+        mkdir -p "\${ED}/usr/share/verilog/\${PN}/"
+        mv Makefile.vcs Makefile
```

Comment:
I think it's better to leave this as "Makefile.vcs" and then do a "make -f Makefile.vcs".